### PR TITLE
Update Adl.cs

### PR DESCRIPTION
**Pull request by m-ghanbari**
اشتباه فاکتور کیری کردین .
ساده سازی فرمول به این شکل اشتباه بوده

PLEASE REMOVE EXPLANATORY TEXT AND UNNEEDED SECTIONS BEFORE CREATING PULL REQUEST.  CREATE A DRAFT PULL REQUEST IF YOU INTEND TO ADD MORE COMMITS.

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation, context, and sources of information or recipes.

Fixes #(issue number)

## Checklist

- [ ] My code follows the existing style, code structure, and naming taxonomy
- [x] I have performed a self-review of my own code and included any verifying manual calculations
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, including INDICATORS.md, README.md, info.xml, etc
- [ ] My changes generate no new warnings and running code analysis does not produce any issues
- [ ] I have added unit tests that prove my fix is effective or that my feature works, and achieves sufficient code coverage
- [x] I have added or run the performance tests that depict execution times similar to others
- [x] New and existing unit tests pass locally with my changes

## Acknowledgements

- [x] I have read and understand [the Apache 2.0 license](https://opensource.org/licenses/Apache-2.0)
- [x] I agree to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

### CODE REVIEW

**Comment by m-ghanbari** [comment]
fix formul a/d

**Comment by DaveSkender** [comment]
The mathematical change you've made does not follow the documented formula.  [ADL](https://daveskender.github.io/Stock.Indicators/indicators/Adl/#content) is a confusing calculation when you compare it to other charting sites because the numbers are often different (see #319).

Please see these references for the formula:
- [Wikipedia](https://en.wikipedia.org/wiki/Accumulation/distribution_index)
- [Investopedia](https://www.investopedia.com/terms/a/accumulationdistribution.asp)
- [ChartSchool](https://school.stockcharts.com/doku.php?id=technical_indicators:accumulation_distribution_line)

File: indicators/Adl/Adl.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public static partial class Indicator
                 int index = i + 1;
 
                 decimal mfm = (h.High == h.Low) ? 0 : ((h.Close - h.Low) - (h.High - h.Close)) / (h.High - h.Low);
-                decimal mfv = mfm * h.Volume;
+                decimal mfv = (h.High == h.Low) ? 0 : ((h.Close - h.Low) - ((h.High - h.Close)* h.Volume)) / (h.High - h.Low);
```
Comment (DaveSkender) [inline indicators/Adl/Adl.cs:35]: `(h.High == h.Low) ? 0 : ((h.Close - h.Low) - (h.High - h.Close))* h.Volume / (h.High - h.Low)`; would make this consistent with the documented formulas; however, that would be mathematically identical to what is already there.

**Comment by DaveSkender** [comment]
- I do not believe these changes are appropriate
- Contributor is unresponsive

**Pull request closed without merging**